README.md

```diff
@@ -1,26 +1,44 @@
-# operator-template
+# MySQL Machine Charmed Operator
 
 ## Description
 
-TODO: Describe your charm in a few paragraphs of Markdown
+The [MySQL](https://www.mysql.com/) operator provides an open-source relational database management system (RDBMS). This repository contains a Juju Charm for deploying MySQL on machines.
+
+This charm is currently in development, with High Availability via Group Replication as a short-term goal.
 
 ## Usage
 
-TODO: Provide high-level usage, such as required config or relations
+To deploy this charm using Juju 2.9.0 or later, run:
 
-## Relations
+```shell
+juju add-model gs
```

Comment:
Should we remark that this should be run under a  !k8s (openstack/lxd/etc) controller?

Reply:
added remark in cc37f2fa62a8ccfb339e75b1fd51ab0c8977f1d7